src/blocks/sound/pulseaudio.rs

```diff
@@ -350,9 +350,10 @@ impl Client {
     }
 
     fn send_update_event() {
-        for tx in &*EVENT_LISTENER.lock().unwrap() {
-            tx.blocking_send(()).unwrap();
-        }
```

Comment:
Was this a clippy lint?

Reply:
No. I don't remember what I did, but this unwrap caused a panic. So instead of panicking this code will remove the listener from the list if channel was closed.

Reply:
Can you make it a separate commit so that we don't accidentally regress in the future?

Reply:
Pushed to master.